conf/config_template.yaml

```diff
@@ -30,6 +32,8 @@ global:
   debug_mode: True
   coordconv_convert: False
   coordvonc_scale:
+  max_used_ram: 2000
+  max_used_perc: 15
```

Comment:
would add a comment (in elsewhere in doc) to tell user what default values are (and of course that these values as useless if num_gpu=0)